### PR TITLE
UNI-197: Limit concurrent models to 2

### PR DESCRIPTION
**Pull request by lscheinkman**
@brev Please review when you have a chance

### CODE REVIEW

**Comment by brev** [comment]
@lscheinkman please merge now if you feel ok about it, I can't review until I'm done with my DB task, may be a bit sorry

**Comment by brev** [comment]
@lscheinkman reviewing now

File: unicorn/frontend/browser/actions/AddModel.js [hunk 1]
```diff
@@ -18,6 +18,7 @@
 // http://numenta.org/licenses/
 
 'use strict';
+import {ACTIONS} from '../lib/Constants';
```
Comment (brev) [inline unicorn/frontend/browser/actions/AddModel.js]: what do you think about putting the Action constants in the shared global config I created?

**Comment by brev** [comment]
:+1: thanks for the IPC example in ModelClient/Server, I might borrow some of this for the DB client/server

File: unicorn/frontend/browser/actions/AddModel.js (hunk 1 above)
Comment (lscheinkman) [inline unicorn/frontend/browser/actions/AddModel.js]: @brev Where is the "global config" file? I can only find the `ConfigServer.js` and `ConfigClient.js`?

Comment (brev) [inline unicorn/frontend/browser/actions/AddModel.js]: sure, I was just thinking of adding them to config, so it's all in a single place. no big deal either way, just 1 less file.

https://github.com/numenta/numenta-apps/blob/master/unicorn/frontend/config/default.json
https://github.com/numenta/numenta-apps/blob/master/unicorn/frontend/config/

thanks

**Pull request merged**